README.md

```diff
@@ -30,6 +30,60 @@ revision: HEAD
 
 See `conf/machine` for the complete list of supported devices.
 
+## Quick build
+Please refer to the [Yocto Project Quick Start Guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html) to set up your Yocto Project build environment.
+
+1. Clone this layer and its dependencies
+	```
+	git clone https://github.com/quic-yocto/meta-qcom-hwe.git -b main
+	git clone git://git.yoctoproject.org/meta-qcom -b master
```

Comment:
we should use/recommend the http link instead of git://
https://git.yoctoproject.org/meta-qcom
https://git.yoctoproject.org/poky